src/presets/service-worker.ts

```diff
@@ -55,6 +49,11 @@ export const serviceWorker = defineNitroPreset(() => {
       preview: 'npx serve ./public'
     },
     hooks: {
+      'prerender:route' (route) {
+        // TODO: use baseURL from nitro options
+        const baseURL = '/'
```

Comment:
@pi0 @danielroe  Is there any way to access `nitroApp` inside presets other that passing it to hook?

Reply:
No. Presets are evaluated for constructing config. We might improve this by supporting lazy evaluated hooks but for now you can directly pass `nireo` as second argument 👍🏼